tests/test_digital.py

```diff
@@ -391,3 +404,140 @@ def test_kron_batch() -> None:
         density_matrices.append(density_matrice)
     dm_expect = torch.cat(density_matrices, dim=2)
     assert torch.allclose(dm_out, dm_expect)
+
+
+@pytest.mark.parametrize("FlipGate", [BitFlip, PhaseFlip, Depolarizing, PauliChannel])
+def test_flip_gates(FlipGate: Noise) -> None:
```

Comment:
Same. I think it can be broken down in fixtures that'll help making the test more readble. We discuss that if you wish.

Reply:
Changed.